exec/java-exec/src/main/java/org/apache/drill/exec/planner/fragment/Materializer.java

```diff
@@ -138,7 +138,7 @@ public PhysicalOperator visitUnnest(UnnestPOP unnest, IndexedFragmentNode value)
     final Wrapper info;
     final int minorFragmentId;
 
-    UnnestPOP unnest = null;
+    Stack<UnnestPOP> unnest = new Stack<>();
```

Comment:
Could you please replace `Stack` usage by `Deque` and `ArrayDeque` and make fields private final